docker/test/scripts/test_env

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+/scripts/setup_env
+source /app/.env
+
+export RAILS_ENV=test
+bundle install --with test development --path vendor/bundle -j 4
+bundle exec rake db:drop db:create db:migrate
```

Comment:
What do you think about removing `db:drop` here? In my tests the migrations took a lot of time, which got a bit annoying in consecutive runs to check if a changed fixed a single spec.
To start from scratch one could always remove the docker containers (or just the database data).